corelib-db/src/main/java/eu/europeana/corelib/db/service/impl/UserServiceImpl.java

```diff
@@ -18,9 +18,9 @@
 import eu.europeana.corelib.definitions.edm.beans.FullBean;
 import eu.europeana.corelib.definitions.edm.entity.Aggregation;
 import eu.europeana.corelib.definitions.edm.entity.Proxy;
+import eu.europeana.corelib.record.RecordService;
 import eu.europeana.corelib.web.exception.EuropeanaException;
 import eu.europeana.corelib.web.exception.ProblemType;
-import eu.europeana.corelib.search.SearchService;
 import eu.europeana.corelib.web.exception.EmailServiceException;
 import eu.europeana.corelib.web.service.EmailService;
 import org.apache.commons.lang3.StringUtils;
```

Comment:
same thing for this -impl class

Reply:
Yeah, I was in doubt whether to preserve the interface+impl combination or not. I decided to keep it, at least for now.

Reply:
Some programmers feel that everything that is public should have an interface. I'm not so hard-core. As far as I can see now this is not used by Metis or other software, so that very much reduces the value of having separate interfaces and implementation. I see no harm in keeping it, but we can get rid of it eventually if you don't like it. Let's remove it then in the next clean-up round.

Reply:
re. "some programmers feel ..." - indeed, I've seen more discussion about such issues than I think they're worth. While it may be essentially a good idea to create interfaces for every public class, it shouldn't be a dogma.